src/Core/Scheduler/Providers/OGCWebServiceHelper.js

```diff
@@ -114,7 +114,11 @@ export default {
                 projection.getCoordWMTS_WGS84(tileCoord, tile.extent, tileMatrixSet);
         }
     },
-    computeTMSCoordinates(tile, extent) {
+    // The topLeft parameter is to be set at true if the computation of the
+    // coordinates needs to be inverted to match the same scheme as OSM, Google
+    // Maps or other system. See link below for more information
+    // https://alastaira.wordpress.com/2011/07/06/converting-tms-tile-coordinates-to-googlebingosm-tile-coordinates/
+    computeTMSCoordinates(tile, extent, topLeft = false) {
```

Comment:
I'd prefer the second argument to be `origin` (or `yOrigin`) with a default value : `top-left` than a boolean.

This way at the caller place it's easier to read the code:
```js
computeTMSCoordinates(tile, extent, layer.origin);
computeTMSCoordinates(tile, extent, 'top-left');
```
Are both clearer than:
```js
computeTMSCoordinates(tile, extent, false);
computeTMSCoordinates(tile, extent, true);
```

Reply:
I don't know, wouldn't setting a string like `top-left` induced that we could have value like `bottom-right` or something else ? But I'm maybe being too picky here ;)

Reply:
`yOrigin: top/bottom`? If we never have a {top,bottom}-right, then it's the clearest option imo.